code/modules/research/designs/autolathe_designs.dm

```diff
@@ -799,3 +799,11 @@
 	materials = list(MAT_METAL = 7500, MAT_GLASS = 100)
 	build_path = /obj/item/pet_carrier
 	category = list("initial", "Misc")
+
+/datum/design/miniature_power_cell
+	name = "Light Fixture Battery"
+	id = "miniature_power_cell"
+	build_type = AUTOLATHE
+	materials = list(MAT_GLASS = 20)
+	build_path = /obj/item/stock_parts/cell/emergency_light
+	category = list("initial", "Electronics")
```

Comment:
wait isn't this from your red light PR?

Reply:
You're looking at the commit that fixed conflicts!